views/biobank/order-quanum.html.twig

```diff
@@ -0,0 +1,116 @@
+{% extends 'base.html.twig' %}
+{% block title %}Biobank {{ participant.biobankId }} - {% endblock %}
+{% block body %}
+    <div class="page-header">
+        <h2><i class="fa fa-medkit" aria-hidden="true"></i> Order Details</h2>
+    </div>
+    <div class="row">
+        <div class="col-sm-4">
+            <dl class="dl-horizontal">
+                {% if is_granted('ROLE_SCRIPPS') or is_granted('ROLE_AWARDEE') %}
+                    <dt>Participant ID</dt>
+                    <dd>{{ participant.id }}</dd>
+                {% endif %}
+                <dt>Biobank ID</dt>
+                <dd>
+                    {% if is_granted('ROLE_SCRIPPS') or is_granted('ROLE_BIOBANK') %}
+                        <a href="{{ path('biobank_participant', { biobankId: participant.biobankId }) }}">{{ participant.biobankId }}</a>
+                    {% else %}
+                        {{ participant.biobankId }}
+                    {% endif %}
+                </dd>
+                <dt>Paired Awardee</dt>
+                <dd>{{ participant.awardee ? app.getAwardeeDisplayName(participant.awardee) : '(not paired)' }}</dd>
+                <dt>Paired Site</dt>
+                <dd>{{ participant.site ? app.getSiteDisplayName(participant.siteSuffix) : '(not paired)' }}</dd>
+            </dl>
+        </div>
+        <div class="col-sm-4">
+            <p class="lead">
+                Order ID: <strong>{{ order.order_id }}</strong>
+            </p>
+            <h3>
+                <i class="fa fa-medkit" aria-hidden="true"></i>
+                {{ order.created_ts|date('n/j/Y', app.userTimezone) }}
+            </h3>
+        </div>
+    </div>
+
+    {% if order.status == 'unlock' %}
```

Comment:
Do we need this part of the view dealing with unlocked/cancelled orders?

Reply:
Removed logic in 0d97e6f